### PR TITLE
Placeholder landing page

### LINKED ISSUE #1: Create landing page
This should be low-effort, since (almost) anything is better than the holding page we have now. See https://github.com/quarkiverse/quarkiverse-docs/pull/94#issuecomment-1617641150 for discussion of the kinds of stuff we'll put here. 

It's a bit more effort than some options, but I propose this as our starting point: 

image

### PR DESCRIPTION
**Pull request by holly-cummins**
Resolves #1.

### CODE REVIEW

**Comment by holly-cummins** [comment]
Hurrah, #2 worked, and we have a preview to review.

File: src/index.html [hunk 1]
```diff
@@ -36,21 +33,20 @@
...
       
 
-      <h1 id="quarkiversegithubio">quarkiverse.github.io</h1>
-<p>Website for quarkiverse.io</p>
+<h1>Welcome to the Quarkiverse</h1>
 
+        <img src="images/quarkiverse-logo.png" width="200" alt="quarkiverse logo"/>
+
+        <div class="row">
+            <a class="button-cta" href="http://docs.quarkiverse.io">Read extension documentation</a>
```
Comment (gastaldi) [inline src/index.html]:
```suggestion
            <a class="button-cta" href="https://docs.quarkiverse.io">Read extension documentation</a>
```

**Comment by holly-cummins** [comment]
The downside of just cloning the existing page in #2 is that it wasn't obvious, post-merge, that the github pages publish isn't actually working. So the landing page is still the original one, not my new shiny one. Will investigate.

**Comment by holly-cummins** [comment]
Phew, #4 has done it. We have a landing page!

**Comment by gastaldi** [comment]
Awesome! Too bad it renders weird in my phone :)

Screenshot_20230706_180810_Chrome

**Comment by gastaldi** [comment]
But no worries, we can figure that out in other iterations 😉

**Pull request merged**